KomaMRICore/src/simulation/Bloch/BlochVoxelDictSimulation.jl

```diff
@@ -0,0 +1,82 @@
+import KomaMRICore: SimulationMethod, initialize_spins_state, mul!, output_Ndim,
+    run_spin_excitation!, run_spin_precession!, sim_output_dim
+
+
+Base.@kwdef struct BlochVoxelDict <: SimulationMethod 
+    N_spins_per_voxel::Int=1
+end
+
+output_Ndim(sim_method::BlochVoxelDict) = 2
```

Comment:
I changed the simulation_method extension works in https://github.com/JuliaHealth/KomaMRI.jl/commit/148a257b955005ad01b5adc266ba1a02c65883a1. This is no longer necessary
```suggestion
```